packages/mask/shared-ui/locales/zh-TW.json

```diff
@@ -447,6 +449,27 @@
     "plugin_collectible_make_an_offer": "商議價格",
     "plugin_collectible_approved_by_open_sea": "通過勾選此框，我確認此收藏品未經 OpenSea 審查。",
     "plugin_collectible_legal_text": "通過勾選此框，我同意 OpenSea 的 <terms>使用者條約</terms>。",
+    "plugin_cryptoartai_description_title": "描述",
+    "plugin_cryptoartai_edition": "版本",
+    "plugin_cryptoartai_operator": "操作員",
+    "plugin_cryptoartai_activity_type": "類型",
+    "plugin_cryptoartai_status": "狀態",
+    "plugin_cryptoartai_time": "時間",
+    "plugin_cryptoartai_price": "價錢",
+    "plugin_cryptoartai_tx": "Tx",
+    "plugin_cryptoartai_latest_bid": "最新出价",
+    "plugin_cryptoartai_description": "最新出价 <strong>{{bidPrice}} {{symbol}}</strong>，价格 <strong>{{price}} {{symbol}}</strong>。版本 {{soldNum}} / {{totalAvailable}} #{{editionNumber}}。",
+    "plugin_cryptoartai_no_price_description": "最新出价 <strong>{{bidPrice}} {{symbol}}</strong>。版本 {{soldNum}} / {{totalAvailable}} #{{editionNumber}}。",
+    "plugin_cryptoartai_sold_description": "已出售價格 <strong>{{soldPrice}}{{symbol}}</strong>。版本 {{soldNum}} / {{totalAvailable}} #{{editionNumber}}。",
+    "plugin_cryptoartai_buy": "購買",
+    "plugin_cryptoartai_buy_now": "立即購買",
+    "plugin_cryptoartai_current_balance": "當前餘額",
+    "plugin_cryptoartai_current_highest_offer": "當前最高報價為 ",
+    "plugin_cryptoartai_bid_least": "你必須至少出價 ",
+    "plugin_cryptoartai_escrowed": "您的報價將被託管在智能合約中，直到被接受或您撤回為止",
+    "plugin_cryptoartai_current_balance_is": "當前餘額為 ",
+    "plugin_cryptoartai_auction_end": "拍賣已結束",
+    "plugin_cryptoartai_auction_end_time": "拍賣結束時間 ",
```

Comment:
We manage i18n translation via [Crawdin](https://crowdin.com/translate/mask-network/41/en-zhcn?filter=basic&value=0). You only need to add `en-US` translation, and leave others to Crawdin

Reply:
Just keep these translation